app/models.py

```diff
@@ -1986,16 +1985,16 @@ class DateTimeDimension(db.Model):
     bst_date = db.Column(db.Date, nullable=False, primary_key=True, index=True)
     year = db.Column(db.Integer(), nullable=False)
     month = db.Column(db.Integer(), nullable=False)
-    month_name = db.Column(db.Text(), nullable=False)
+    month_name = db.Column(db.String(), nullable=False)
     day = db.Column(db.Integer(), nullable=False)
     bst_day = db.Column(db.Integer(), nullable=False)
     day_of_year = db.Column(db.Integer(), nullable=False)
-    week_day_name = db.Column(db.Text(), nullable=False)
+    week_day_name = db.Column(db.String(), nullable=False)
```

Comment:
Same question about string/text. Weekdays have set max lengths. No need for infinite here.